pom.xml

```diff
@@ -35,7 +35,7 @@
         <image.name>ghcr.io/idealo/${project.artifactId}:${project.version}</image.name>
 
         <kotlin.code.style>official</kotlin.code.style>
-        <kotlin.version>1.8.22</kotlin.version>
+        <kotlin.version>1.9.22</kotlin.version>
```

Comment:
do not merge. update progress is tracked here: https://github.com/idealo/spring-endpoint-exporter/pull/127